We do not have these optional test dependencies packaged yet.

--- lazy_object_proxy-1.12.0/tox.ini.orig
+++ lazy_object_proxy-1.12.0/tox.ini
@@ -32,9 +32,6 @@
 deps =
     pytest
     pytest-benchmark
-    Django
-    objproxies==0.9.4
-    hunter
     setuptools>=80
     setuptools_scm>=8
     cover: pytest-cov
